### ISSUE TITLE
purge form from browser db during logout

### ISSUE THREAD

**Issue by MartijnR** [enketo/enketo#1127]
Use case:

1. user A loads offline-capable form, enters data, submits et
2. user A logs out
3. user B works on same device/browser, and opens form

expected: login screen
experienced: it loads the form (from the cache, and soon after receives a 401 so submitting is not possible

4. Only after user B reloads again (after the 401 was received in the previous session), is the login screen shown.

* [ ] A pragmatic, easy, and I think good solution would be to purge the form during logout.

However, this use case brings up another issue. What should we do with any draft/final records that are present in the browser db at the time user A logs out?

* [ ] do something with records during logout - tbc
